mixer/pkg/protobuf/yaml/dynamic/handler.go

```diff
@@ -399,6 +404,102 @@ func (s Svc) encodeRequest(qr proto.Marshaler, dedupID string, encodedInstances
 	return ba, nil
 }
 
+// CheckOutput is a generic container for the response of remote check calls with output.
+// It is used to unpack the value using a combination of the static and dynamic decoders.
+type CheckOutput struct {
+	result    *v1beta1.CheckResult
+	output    *attribute.MutableBag
```

Comment:
Done. This happened in a few places. A later refactor could probably a new mutable bag that has an automatic prefix, to avoid passing both the bag and the string in several places.